.github/workflows/ci-pr.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         maven-version: 3.8.8
         maven-mirror: 'https://dlcdn.apache.org/maven/maven-3/'
 #    - name: Login dockerhub
```

Comment:
We can remove these lines.

Reply:
Once we figure out how to configire the repo so that only PRs by contributors will execute and allow access to secrets, we will uncomment the docker login